### PR TITLE
fix: add apple for cors

### PR DESCRIPTION
**Pull request by markusahlstrand**
I guess this check shouldn't be throwing..

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -66,6 +67,7 @@ app.use(loggerMiddleware);
 app.get("/", async (ctx: Context<{ Bindings: Env; Variables: Var }>) => {
   return ctx.json({
     name: packageJson.name,
+    foo: "bar",
```
Comment (danielgent) [inline src/app.ts:70]: Eh?

File: src/app.ts [hunk 2]
```diff
@@ -34,6 +34,7 @@ const ALLOWED_ORIGINS = [
   "https://auth-admin.sesamy.dev",
   "https://login2.sesamy.com",
   "https://auth-admin.sesamy.com",
+  "https://appleid.apple.com",
```
Comment (danielgent) [inline src/app.ts:37]: Lol right. Yes this exception message is absolutely wrong then!

File: src/app.ts (hunk 1 above)
Comment (markusahlstrand) [reply src/app.ts:70]: Oops.. Just checking that you pay attention :)

**Review by danielgent — APPROVED** [review]

File: src/app.ts (hunk 2 above)
Comment (markusahlstrand) [reply src/app.ts:37]: Yeah, it's not great. Let patch fix this properly in a new pr

**Pull request merged**